Source/Scene/Cesium3DTileStyleEngine.js

```diff
@@ -0,0 +1,85 @@
+/*global define*/
+define([
+        '../Core/defined',
+        '../Core/defineProperties'
+    ], function(
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function Cesium3DTileStyleEngine() {
+        this._style = undefined;      // The style provided by the user
+        this._styleDirty = false;     // true when the style is reassigned
+        this._lastStyleTime = 0;      // The "time" when the last style was assigned
```

Comment:
I'm not really sure what to call this. It's like a frame marker but can be reset or changed.